src/HH4b/postprocessing/postprocessing.py

```diff
@@ -344,15 +339,8 @@ def correct_mass(events_dict, mass_str):
         ),
     }
 
-    if txbb_version == "glopart-v2":
-        correct_mass(events_dict_nosyst, mass_str)
-        correct_mass(events_dict_syst_bg, mass_str)
-        correct_mass(events_dict_syst_sig, mass_str)
-
     if scale_and_smear:
-        add_rawmass(events_dict_nosyst, mass_str)
-        add_rawmass(events_dict_syst_bg, mass_str)
-        add_rawmass(events_dict_syst_sig, mass_str)
+        # re-run scaling and smearing of mass variables
         events_dict_syst_bg = scale_smear_mass(events_dict_syst_bg, year, mass_str)
```

Comment:
Is this needed? I guess that for the new ntuples scale_and_smear will be set to False?

Reply:
yeah, exactly... for the new ntuples, we wouldn't need to run it (unless we update the SFs, etc.)

in principle, it would be good to check if we re-run it with the same SFs, we get the same templates (approximately)